include/villas/nodes/delta_share/Protocol.h

```diff
@@ -0,0 +1,210 @@
+
+#ifndef __PROTOCOL_H__
+#define __PROTOCOL_H__
+
+#include <iostream>
+#include <nlohmann/json.hpp>
+#include <map>
+#include <string>
+#include <vector>
+#include <boost/optional.hpp>
+#include <boost/optional/optional_io.hpp>
+
+namespace DeltaSharing
+{
+
+    namespace DeltaSharingProtocol
+    {
+
+        using json = nlohmann::json;
```

Comment:
Please do not introduce a second JSON library into the code-base.

We are already using libjansson throughout VILLASnode.